README.md

```diff
@@ -34,6 +34,7 @@ __NOTE__: It's early days so please be gentle when reporting issues :) As author
 
         post 'creates a new blog' do
           consumes 'application/json'
+          operation_description 'Detailed implementation notes for the create a new blog API endpoint'
```

Comment:
Could we just go with "notes" on this one. I really want it to be "description" so that it reflects the terminology in the swagger-spec itself. Given that's not possible, I could settle on succinct alternative such as "notes".